packages/webpack-cli/lib/groups/ConfigGroup.js

```diff
@@ -42,6 +42,16 @@ const getDefaultConfigFiles = () => {
 
 const getConfigInfoFromFileName = (filename) => {
     const fileMetaData = parse(filename);
+    // .cjs is not available on interpret side, handle it manually for now
```

Comment:
Can you clarify?

Reply:
@evilebottnawi check https://github.com/gulpjs/interpret/blob/57cd5973c313c420b1ad24a922d1c95ebec3c532/index.js#L181 doesn't have `.cjs` hence need manual handling